### PR TITLE
support init graph using tensorflow model in python

### PR DESCRIPTION
**Pull request by yangw1234**
## What changes were proposed in this pull request?

add tensorflow load method

## How was this patch tested?

manually tested

### CODE REVIEW

**Comment by yangw1234** [comment]
pr validation pass

File: docs/docs/APIdocs/Model/Functional.md [hunk 1]
```diff
@@ -138,4 +138,40 @@ add = CAddTable()(relu1, relu2)
...
+    
+    bigdl_model = Model(input, output)
+    bigdl_result = bigdl_model.forward(tensor)
+
+    print("Tensorflow forward result is " + str(tensorflow_result))
+    print("BigDL forward result is " + str(bigdl_result))
+
+    np.testing.assert_almost_equal(tensorflow_result, bigdl_result, 6)
+    print("The results are almost equal in 6 decimals")
+```
```
Comment (jason-dai) [inline docs/docs/APIdocs/Model/Functional.md]: Move the doc to Tensorflow support, instead of functional API

Comment (jason-dai) [inline docs/docs/APIdocs/Model/Functional.md]: I don't think the example matches your API below

File: pyspark/bigdl/nn/layer.py [hunk 2]
```diff
@@ -374,15 +376,36 @@ class Model(Container):
...
+     or "big_endian") and node_type parameter ("bigdl" or "tensorflow")
+    node_type parameter.
     """
 
     def __init__(self,
                  inputs,
                  outputs,
-                 bigdl_type="float"):
-        super(Model, self).__init__(None, bigdl_type,
+                 bigdl_type="float", byte_order="little_endian", node_type="bigdl"):
```
Comment (jason-dai) [inline pyspark/bigdl/nn/layer.py]: change "node_type" to "model_type"

File: pyspark/bigdl/nn/layer.py [hunk 3]
```diff
@@ -374,15 +376,36 @@ class Model(Container):
...
+        else:
+            from bigdl.util.tf_utils import generate_graph_file
+            path, input_names, output_names = generate_graph_file(to_list(inputs), to_list(outputs))
+            super(Model, self).__init__(None, bigdl_type,
+                                        path + '/model.pb',
+                                        input_names,
+                                        output_names,
+                                        byte_order)
+            try:
+                shutil.rmtree(path)
```
Comment (jason-dai) [inline pyspark/bigdl/nn/layer.py]: Why not handle this in tf_utils?

**Comment by yiheng** [comment]
LGTM

**Comment by yangw1234** [comment]
pr validation pass

**Pull request merged**